test/verify/check-metrics

```diff
@@ -772,8 +772,13 @@ class TestCurrentMetrics(MachineCase):
         # allocate a chunk of memory; this may cause other stuff to get unmapped,
         # thus not exact addition, but usage should go up
         size = 300 if have_swap else 200
-        m.execute("systemd-run --collect --slice cockpittest --unit mem-hog awk "
-                  """'BEGIN { x = sprintf("%""" + str(size) + """000000s",""); system("touch /tmp/hogged; sleep infinity") }'""")
+        m.write("/run/memhog.sh", f"""#!/usr/bin/awk -f
```

Comment:
FTR: This uses /run instead of /tmp, as otherwise SELinux throws a spanner into the works again, and doesn't let systemd-run execute the script.

Reply:
Move to /usr/local/bin/, as /run doesn't work for Debians: that's mounted `noexec` there.  Also fixed to clean up after itself.